metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -629,23 +785,26 @@ def calculate_lineage_for_project(
     def get_lineage_for_table(
         self,
         bq_table: BigQueryTableRef,
+        bq_table_urn: str,
```

Comment:
Doesn't have to be done in this PR, but I'd like to pass into the lineage extractor `dataset_urn_builder: Callable[[BigQueryTableRef], str]` like I do with the bq usage extractor, so that we always use the same method to generate dataset urns. Then, I think this won't be necessary (and we can remove the mce_builder call below in favor of it too)